features/step_definitions/tag_steps.rb

```diff
@@ -226,6 +230,22 @@
   blank_tag.save!(validate: false)
 end
 
+Given "I have just created the canonical media tag {string}" do |name|
```

Comment:
Let's use "I create" / "I recategorize", most of our step names use present tense.